mu/integration_tests/test_pca9539r.py

```diff
@@ -2,28 +2,32 @@
 import time
 
 from mu.integration_tests import int_test
-from mu.sims.pca9539r import Pca9539r, NUM_PCA_PINS
+from mu.sims.front_pd import FrontPd
+from mu.sims.sub_sims.pca9539r import NUM_PCA9539R_PINS
+from mu.protogen import stores_pb2
+
+# 0x74 is address used in smoketest
+PCA9539R_KEY = (stores_pb2.MuStoreType.PCA9539R, 0x74)
```

Comment:
Perhaps there should be a helper function in the pca9539r sim to construct pca9539r keys from i2c addresses (mostly so there's a bit of documentation on what the second element of the key is).